tests/tasks/test_blended_skill_talk.py

```diff
@@ -402,5 +402,31 @@ def test_safe_personas(self):
             self.assertEqual(len(personas), count)
 
 
+class TestBlendedSkillTalkModels(unittest.TestCase):
```

Comment:
This should go in the night/gpu tests folder, not the tasks folder.

Reply:
Makes sense - moved